src/main/java/org/folio/circulation/domain/representations/logs/CirculationCheckInCheckOutLogEventMapper.java

```diff
@@ -0,0 +1,153 @@
+package org.folio.circulation.domain.representations.logs;
+
+import static java.util.Optional.ofNullable;
+import static org.folio.circulation.support.json.JsonPropertyWriter.write;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.folio.circulation.domain.CheckInContext;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.domain.LoanAndRelatedRecords;
+import org.folio.circulation.domain.UpdatedRequestPair;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CirculationCheckInCheckOutLogEventMapper {
+
+  public static final String CHECK_IN = "CHECK_IN_EVENT";
+  public static final String CHECK_OUT = "CHECK_OUT_EVENT";
+
+  public static final String PL_LOG_EVENT_TYPE = "logEventType";
```

Comment:
What does the `PL` prefix mean?

Reply:
PL is just prefix to know that it is field of payload.

Reply:
Ah ok. And not all of the constants are for payload properties?

If that is the case, I suggest maybe extracting either a class for the payload constants or a separate mapper for payload itself.